config-connector/solutions/projects/kpt/owned-project/README.md

```diff
@@ -0,0 +1,55 @@
+Owned Project
+==================================================
+
+# NAME
+
+  owned-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in a folder, binding an IAM member
+  as project owner.
+
+# CONSUMPTION
+
+  Fetch the package using [kpt](https://googlecontainertools.github.io/kpt/).
+
+  `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/projects/kpt/owned-project owned-project`
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a
+  service account with the following roles in
+  the folder:
+  - roles/resourcemanager.folderViewer
+  - roles/resourcemanager.projectCreator
+  - roles/iam.securityAdmin
+
+# USAGE
```

Comment:
Should we have the setters organized in a table? (I thought this is what we decided to do)

Reply:
We definitely decided to move in that direction, but since support for outputting setter markdown tables still hasn't made it into an official kpt release it might be too soon to include them in incoming pull requests.
I think we're already planning to go back and standardize solution READMEs after we get more merged.